### PR TITLE
Adding recipe for fett

### PR DESCRIPTION
**Pull request by tyrius02**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).

### CODE REVIEW

File: recipes/fett/LICENSE.txt [hunk 1]
```diff
@@ -0,0 +1,21 @@
+MIT License
```
Comment (dopplershift) [inline recipes/fett/LICENSE.txt:1]: Where did this license come from? I can't seem to find this in the upstream repository.

Comment (tyrius02) [reply recipes/fett/LICENSE.txt:1]: Wasn't sure quite what to do about the license. The PyPi metadata (https://pypi.org/pypi/fett/json) says MIT, but there's no license file, so I faked it. Set the year to the first commit in the source repo and added the original author to it.

Not the right thing to do? Is it possible to just set `license: MIT` w/o a `license_file`? Honestly, didn't try that.

Comment (dopplershift) [reply recipes/fett/LICENSE.txt:1]: The right first step is to open an issue on the upstream GitHub issue and ask for some clarification (i.e. says "MIT" but there's no license file). IANAL, but I'm really not sure if we can take a minor metadata declaration of "MIT" as a legal document allowing us to distribute code. And without that, in the US, the author(s) retain all rights.

Comment (tyrius02) [reply recipes/fett/LICENSE.txt:1]: Yeah, makes sense. I've opened an issue w/ the upstream repo to add a license file. The last modification to the repo was a year ago, but maybe I'll get lucky and the owner will respond to my issue.

**Pull request closed without merging**